config.yml.example

```diff
@@ -1,10 +1,10 @@
 # Github Personal Access Token (https://github.com/settings/tokens) with rights:
```

Comment:
it is better to use `example` suffix not at the end of filename to allow IDE detect and highlight code syntax correctly.

`config.example.yml` would be great.

Reply:
Thanks for suggestion.